pyscript.core/tests/integration/test_02_display.py

```diff
@@ -87,6 +92,7 @@ def test_tag_target_attribute(self):
         goodbye = self.page.locator("#goodbye")
         assert goodbye.inner_text() == "goodbye world"
 
+    @skip_worker("NEXT: display target does not work properly")
```

Comment:
`<script>` can't be a target ... ew talked about it and it's a *display* issue we need to fix, not a library issue ... there was a discussion around this, remember? if the ID is a `<script>` tag, then get its `target.id` instead ... this warning is misleading to me.